### ISSUE TITLE
Navigation Bar Has Inconsistent Spacing

### ISSUE THREAD

**Issue by tywrenn** [openemr/openemr#5723]
Hey 🙂 It's been awhile since I've looked at the UI of this, but it seems like the navigation bar has inconsistent spacing throughout all themes you switch through. Were we trying to get rid of some spacing on the navigation bar at all? If so, we did it incorrectly. Please advise on what was done as I might make some fixes to this in my free time.

**Comment by sjpadgett** [comment]
Hey @tywrenn , 
Most of the demographics was converted to twig by Robert. We shorten some menu titles, moved some however, I'm not sure much attention was paid to menu spacing.
Even so, yes work needs to be done where padding and margins are too much.

We'd welcome any fixes you care to share and hope you are well.

**Comment by tywrenn** [comment]
After reviewing some things, here's what I found:

- Issue #5608 's fix did away with a key file called `default-variables`, only in the `style_light` theme. These variables aren't what you think they are. They actually go across every OpenEMR theme and set default variables for Bootstrap to import in. Removing these can cause theme issues relating to colors and Bootstrap-specific settings. Changing the gray classes are fine since they aren't defaults but getting rid of this file caused a lot of theme issues. I'll be handling this in my upcoming PR.
- PR  #4626 Tab UI improvement is where the issues started overall. `.menuLabel` was added to the `style_light` where it isn't supposed to be at. It should have been put in the `tabs_style_full` or `tabs_style_compact` code. A big issue I have with this PR is it also says that it reduces the size of the navigation bar, yet doesn't seem to address the fact that we have a compact and full style navigation option selector. If we wanted to get rid of that option we could, yet this isn't the way to do it. I'll be correcting this PR and moving the menu label code to the compact nav for now until we figure this out. I'll also be moving the other nav corrections to a more appropriate location.
